src/main/webapp/app/entities/programming-exercise/status/programming-exercise-instructor-status.component.html

```diff
@@ -0,0 +1,14 @@
+<fa-icon
+    id="instructor-status-template"
+    icon="exclamation-triangle"
+    class="text-warning"
+    *ngIf="participationType === 'TEMPLATE' && result && result.score > 0"
```

Comment:
Can you use the enum value instead of a string so that we would get a compiler error in case of a typo?
You would need to make the enum value TEMPLATE available in the corresponding component.ts

Reply:
You are right, I changed that.